lib/matplotlib/axes/_axes.py

```diff
@@ -2011,6 +2011,19 @@ def step(self, x, y, *args, where='pre', data=None, **kwargs):
         kwargs['drawstyle'] = 'steps-' + where
         return self.plot(x, y, *args, data=data, **kwargs)
 
+    @staticmethod
+    def _convert_dx(x0, x, dx, convert):
+        """ Small helper to do logic of width conversion flexibly """
```

Comment:
Minor thing:

```suggestion
        """Small helper to do logic of width conversion flexibly."""
```
Single line docstrings do not have leading or trailing spaces.